substrate/frame/staking/src/mock.rs

```diff
@@ -223,6 +225,19 @@ impl pallet_bags_list::Config<VoterBagsListInstance> for Test {
 	type Score = VoteWeight;
 }
 
+impl pallet_staking_async_ah_client::Config for Test {
```

Comment:
I would leave staking classic entirely in its old state, without being ever tweaked to work with AH-client?